lib/routes/auth.js

```diff
@@ -47,6 +47,8 @@ router
   })
 
   .post('/signin', ({ body }, res, next) => {
+    console.log(body);
```

Comment:
This probably shouldn't go to production?

Reply:
True that! Thanks for the extra pair of eyes. It's fixed